Week-5/Assignment-4/src/calculator.test.js

```diff
@@ -0,0 +1,47 @@
+const mathOperations = require("./calculator");
+
```

Comment:
Add more test cases

Reply:
Thanks @sairamin, added more test cases, please review and merge